com.unity.render-pipelines.universal/Runtime/UniversalRenderPipeline.cs

```diff
@@ -302,6 +302,11 @@ static bool TryGetCullingParameters(CameraData cameraData, out ScriptableCulling
             if (cameraData.xr.enabled)
             {
                 cullingParams = cameraData.xr.cullingParams;
+
+                // Sync the FOV on the camera to match the projection from the XR device
+                if (!cameraData.camera.usePhysicalProperties)
```

Comment:
Do you think we might need to do this path in other places? If so maybe add an internal utility function to do it?